src/components/ChatLog.jsx

```diff
@@ -0,0 +1,38 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+/* added sender, body, and timeStamp props*/
+const ChatLog = ({ entries, onToggleLike }) => {
+  return (
+    <div className="chat-log">
+      {entries.map((entry) => (
+        <ChatEntry
+          key={entry.id}
```

Comment:
Nice use of the `entry.id` for the `key` values since our messages have unique ids!